tests/CLI/CLI.h

```diff
@@ -55,4 +55,13 @@ struct CompleteIncompleteStruct;
 typedef struct IncompleteStruct IncompleteStruct;
 
 DLL_API IncompleteStruct* createIncompleteStruct();
-DLL_API void useIncompleteStruct(IncompleteStruct* a);
\ No newline at end of file
+DLL_API void useIncompleteStruct(IncompleteStruct* a);
+
+#ifdef _WIN32
+void testMSStyleAssembly() {
```

Comment:
Move { to the next line as per our convention.

Reply:
Do you guys have a document describing your conventions ? It would certainly help.

Reply:
We don't, ashamed as I am to admit it. However, in 95 % of all cases it's enough to look at already existing code.